main.py

```diff
@@ -1,10 +1,44 @@
+import json
 import init_django_orm  # noqa: F401
-
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json") as p:
+        players = json.load(p)
+
+    for player_name, player_data in players.items():
+        player_bio = player_data["bio"]
+        player_email = player_data["email"]
+        player_race_data = player_data["race"]
+
+        player_race, created = Race.objects.get_or_create(
+            name=player_race_data["name"],
+            description=player_race_data["description"]
+        )
+
+        for player_race_skill in player_data["race"]["skills"]:
+            skill, created = Skill.objects.get_or_create(
+                name=player_race_skill["name"],
+                bonus=player_race_skill["bonus"],
```

Comment:
Ensure that the `bonus` field in the JSON data is of the correct type if the `Skill` model's `bonus` field is intended to store numerical values. Consider changing the model's field type to `DecimalField` or `IntegerField` if necessary.